server/app/views/admin/questions/QuestionsListView.java

```diff
@@ -105,49 +111,31 @@ private DivTag renderSummary(ActiveAndDraftQuestions activeAndDraftQuestions) {
         .withClasses(Styles.FLOAT_RIGHT, Styles.TEXT_BASE, Styles.PX_4, Styles.MY_2);
   }
 
-  /** Renders the full table. */
-  private Pair<TableTag, ImmutableList<Modal>> renderQuestionTable(
+  /**
+   * @return a pair. First element is list of DOM elements, each representing a row for a particular
+   *     question. Second element is collection of all modals that can be invoked from question
+   *     rows.
+   */
+  private Pair<ImmutableList<DomContent>, ImmutableList<Modal>> renderAllQuestionRows(
```

Comment:
As part of #3359, we'll want to attach a reference class to the container for all of the rows (https://github.com/civiform/civiform/pull/3359/files#diff-af6ae91491683e0fd29d6daa311d8748e93b651f7b67b4521e723b0090c4ca45R139). What do you think about having this return a DivTag rather than an ImmutableList<DomContent> in order to facilitate that? If you'd prefer to wait for the other PR to be merged, that works as well.

Reply:
Done.